Sources/TerminalAPIKit/Models/Operation.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Operation.swift
+//  TerminalAPIKit
+//
+//  Created by Andrew Gates on 8/8/23.
+//
+
+import Foundation
+
+public final class Operation: Codable {
+
+    /// the type of request after which the waiting screen shows: InputRequest, DisplayRequest, and Payment
+    public let type:OperationType
+    
+    /// For now, the Operation.Variant can only have WaitingScreen as value. This shows the waiting screen animation on the terminal after every request until you end the session.
+    public let variant:String
+    
+    /// UserInterfaceData.WaitingScreenTimeoutMs: how long the waiting screen is shown on the terminal display, in milliseconds.
```

Comment:
we probably don't need to mention the field name, just an explanation is enough, e.g.
```swift
     /// UserInterfaceData.WaitingScreenTimeoutMs: how long the waiting screen is shown on the terminal display, in milliseconds.
```

Reply:
For reference, I just copy comments from the Terminal API docs: https://docs.adyen.com/point-of-sale/shopper-engagement/create-session/

Reply:
apologies for this. seems like i forgot to write my own suggestion for the comment. what i meant was that we should not mention the names of the fields in the comments, since they don't match the actual swift field names.
We can simply mention that `userInterfaceData` contains UI parameters related to the operation.